README.md

```diff
@@ -216,6 +216,8 @@ $data = $pdf->getData();
 // Get form data fields
 $pdf = new Pdf('/path/my.pdf');
 $data = $pdf->getDataFields();
```

Comment:
@bravadomizzou Maybe add another example first like `$name = $data['name']` to demonstrate the most obvious use as array?